sonic-xcvrd/xcvrd/sff_mgr.py

```diff
@@ -34,6 +34,9 @@ def log_warning(self, message):
     def log_error(self, message):
         self.logger.log_error("{}{}".format(self.SFF_LOGGER_PREFIX, message))
 
+    def log_debug(self, message):
+        self.logger.log_debug("{}{}".format(self.SFF_LOGGER_PREFIX, message))
```

Comment:
Could we extract this definition into it's own PR to expedite fixing https://github.com/sonic-net/sonic-platform-daemons/issues/605